omnigibson/scenes/scene_base.py

```diff
@@ -426,29 +532,24 @@ def _add_object(self, obj):
         """
         pass
 
-    def add_object(self, obj, register=True, _is_call_from_simulator=False):
+    def add_object(self, obj):
```

Comment:
@cgokmen we discussed how this new paradigm breaks the import / remove object API that used to exist in simulator. Now the scene is the class that imports an object while the simulator is the one that removes the object, which is weird. Can we unify both methods to exist in the same class? Either scene or simulator. IMHO simulator makes a bit more sense based on our (@hang-yin , @ChengshuLi ) discussion but happy to discuss!